deploy/helm/sumologic/templates/metrics/collector/otelcol/opentelemetrycollector.yaml

```diff
@@ -0,0 +1,57 @@
+{{- if and (eq (include "metrics.otelcol.enabled" .) "true") .Values.sumologic.metrics.collector.otelcol.enabled }}
+apiVersion: opentelemetry.io/v1alpha1
+kind: OpenTelemetryCollector
+metadata:
+  name: {{ template "sumologic.metadata.name.metrics.collector.opentelemetrycollector" . }}
+  labels:
+    {{- include "sumologic.labels.metrics.opentelemetrycollector" . | nindent 4 }}
+    {{- include "sumologic.labels.common" . | nindent 4 }}
+spec:
+  mode: statefulset
+  replicas: 3
+  serviceAccount: {{ template "sumologic.metadata.name.metrics.collector.serviceaccount" . }}
+  targetAllocator:
```

Comment:
Does the operator create a target allocator deployment for us?

Reply:
Yes.